contentcuration/contentcuration/frontend/channelEdit/components/ContentNodeOptions.vue

```diff
@@ -1,44 +1,50 @@
 <template>
 
-  <VList>
-    <VListTile v-if="isTopic && canEdit" @click="newTopicNode">
-      <VListTileTitle>{{ $tr('newSubtopic') }}</VListTileTitle>
-    </VListTile>
-    <VListTile v-if="canEdit && !hideEditLink" :to="editLink" @click="trackAction('Edit')">
-      <VListTileTitle>
-        {{ isTopic ? $tr('editTopicDetails') : $tr('editDetails') }}
-      </VListTileTitle>
-    </VListTile>
-    <VListTile v-if="!hideDetailsLink" :to="viewLink" @click="trackAction('View')">
-      <VListTileTitle>{{ $tr('viewDetails') }}</VListTileTitle>
-    </VListTile>
-    <VListTile v-if="canEdit" @click.stop="moveModalOpen = true">
-      <VListTileTitle>{{ $tr('move') }}</VListTileTitle>
-      <MoveModal
-        v-if="moveModalOpen"
-        ref="moveModal"
-        v-model="moveModalOpen"
-        :moveNodeIds="[nodeId]"
-        @target="moveNode"
-      />
-    </VListTile>
-    <VListTile v-if="canEdit" @click="duplicateNode()">
-      <VListTileTitle>{{ $tr('makeACopy') }}</VListTileTitle>
-    </VListTile>
-    <VListTile @click="copyToClipboard()">
-      <VListTileTitle>{{ $tr('copyToClipboard') }}</VListTileTitle>
-    </VListTile>
-    <VListTile v-if="canEdit" @click="removeNode()">
-      <VListTileTitle>{{ $tr('remove') }}</VListTileTitle>
-    </VListTile>
-  </VList>
+  <div
+    style="max-height: 80vh"
+  >
+    <VList ref="optionsList">
+      <template
+        v-for="(group, groupIndex) in groupedOptions"
+      >
+        <VListTile
+          v-for="(option, index) in group"
+          :key="groupIndex + '-' + index"
+          class="options-list-item"
+          ripple
+          :to="option.to"
+          tabindex="0"
```

Comment:
The [mdn web docks](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) recommends that only 0 and -1 be used as tabindex values. I still tried using the sequential positive integer. But when setting values above 0, it only worked with the first option in the menu, before going to the next options, priority was given to all the other elements within the screen that had a tabindex of 0 before going to the second option in the menu since it had a tabindex of 1.

To eliminate the lines of code you suggest, I just let the tabIndex of 0 do the magic and let the browser take care of seeing the next element to select, and I renamed the method to `checkTabBoundaries` since it would no longer fulfill the function of handle the tab, but only check the behavior of the first and last option.

Reply:
`tabindex="0"` is just fine!